index.html

```diff
@@ -1,50 +1,52 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Has funny quotes... maybe">
     <title>Kaashout's webpage</title>
     <meta content="Kaashout's webpage" property="og:title" />
     <meta content="Has funny quotes... maybe" property="og:description" />
     <meta content="https://img.kaashout.tech" property="og:url" />
-    <meta content="https://niasarchive.rooray.xyz/assets/images/gallery01/b3854f51.jpg" property="og:image" />
-
+    <meta content="/saved/T_racist.jpg" property="og:image" />
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+    <h1>10/10 links that should be saved</h1>
+
+    <!-- this is abysmal someone please save this atrocity -->
+    <a href="/saved/ENJOY_YOUR_FOOD.png">Enjoy your food</a>
```

Comment:
@RooRay  If we're going to use images, why not use img tags instead of plain `a` tags?

Reply:
my ~~shit~~ inability to code used `a` instead of img tags

Reply:
and using a tags doesnt embed the image (which is the point)

Reply:
> @RooRay If we're going to use images, why not use img tags instead of plain `a` tags?

no, https://discord.com/channels/324683287165403157/399719720502165515/1163244702590578708 and https://discord.com/channels/324683287165403157/399719720502165515/1163244746148433990